icu4j/main/classes/core/src/com/ibm/icu/util/NoUnit.java

```diff
@@ -10,44 +10,34 @@
  * @draft ICU 60
  * @provisional This API might change or be removed in a future release.
  */
-public class NoUnit extends MeasureUnit {
-    private static final long serialVersionUID = 2467174286237024095L;
-
+public final class NoUnit {
     /**
      * Constant for the base unit (dimensionless and no scaling).
      *
+     * Since ICU 68, this constant equals null.
```

Comment:
A change like this should get the API moved to `@draft ICU 68`.

For PERCENT and PERMILLE I guess it's debatable.

Reply:
I have an open question about this in the ICU 68 API Proposal Status doc.